services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -104,11 +88,7 @@ export const ChoiceListField = ({
     });
   };
 
-  const autosaveContainsParentChoice = (value: string) =>
-    lastAutosaveValue &&
-    lastAutosaveValue.some((autosave) => autosave.value === value);
```

Comment:
Note: I don't think we'll need this anymore because of the new `getNestedFields` function.